README.md

```diff
@@ -15,23 +15,23 @@ Check out the example Shine app above for help getting started.
 Shine SDK requires iOS 7.1+
 
 ###Cocoapods Installation###
-Shine can be installed using [Cocoapods](http://cocoapods.org). Simple add the following to your Podfile and run _pod install_.
+Shine can be installed using [Cocoapods](http://cocoapods.org). Simple add the following to your Podfile and run `pod install`.
 ```bash
 
-pod 'Shine', '~>1.0'
+pod 'Shine', '~>1.1'
 ```
 
 ###Manual Installation###
 
-Shine SDK needs to be installed manually.
+Shine SDK can also be installed manually.
 
- 1. Download the latest shine from [Github](https://github.com/SoundwaveApp/Shine-iOS/tree/master/Shine.framework)
+ 1. Download the latest Shine framework from [Github](https://github.com/SoundwaveApp/Shine-iOS/tree/master/Shine.framework)
  2. Drag Shine.framework into your Xcode project, ticking the “Copy items if needed” checkbox.
```

Comment:
Possibly change this url to stop at Shine-iOS? I know brian did it but seems weird that it links you to a page where there's no download option.